app/helpers/employers_helper.rb

```diff
@@ -0,0 +1,2 @@
+module EmployersHelper
```

Comment:
If this is not needed, I don't think it should be added.

I prefer going with adding as minimal code needed to complete a feature rather than thinking that we'd use it in the future.

YAGNI: You ain't gonna need it
KISS: Keep it simple 😶